src/xnnpack/subgraph.h

```diff
@@ -205,13 +205,16 @@ enum xnn_shape_inference_status {
   xnn_shape_inference_status_no_change,
   // Shape inference change some dimension.
   xnn_shape_inference_status_changed,
```

Comment:
Just curious, would there be a potential use for this state?

Reply:
This is an artefact from the original shape inference prototype.  I think it has been superseded by  xnn_shape_inference_status_reallocation_required

Reply:
shape_inference status is no longer used